recipes/perl-go-perl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "perl-go-perl" %}
+{% set version = "0.15" %}
+{% set sha256 = "423d26155ee85ca51ab2270cee59f4e85b193e57ac3a29aff827298c0a396b12" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: go-perl-0.15.tar.gz
+  url: https://cpan.metacpan.org/authors/id/C/CM/CMUNGALL/go-perl-0.15.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl
+    - perl-data-dumper
+    - perl-extutils-makemaker
+    - perl-data-stag
+
+  run:
+    - perl
+    - perl-data-dumper
+    - perl-data-stag
+
+test:
+  imports:
+    - GO::Parser
+  commands:
+    - which map2slim >& /dev/null
+
+about:
+  home: http://metacpan.org/pod/go-perl
+  license: Perl
```

Comment:
I could not find a real license, which is a problem, do you have more luck?

Reply:
If you look at http://metacpan.org/pod/go-perl it says `unknown` (?)

Reply:
I think we should say unknown then.